cli/src/semgrep/app/scans.py

```diff
@@ -368,14 +370,11 @@ def report_findings(
             logger.info(
                 f"Would have sent complete blob: {json.dumps(complete.to_json(), indent=4)}"
             )
-            return (False, "")
+            return (True, False, "")
```

Comment:
ScanResult(success = True, block_scan = False, block_reason = "") would be more readable. I know we want to get rid of this python code, still ...

Reply:
done